core/templates/pages/contributor-dashboard-page/contributions-and-review/contributions-and-review.component.ts

```diff
@@ -536,6 +544,11 @@ export class ContributionsAndReview
       .reloadOpportunitiesEventEmitter.emit();
   }
 
+  computeTranslationWordCount(translationHtml: string): number {
+    return this.htmlLengthService.
+      computeHtmlLengthInWords(translationHtml);
+  }
```

Comment:
Actually this method doesn't seem necessary. Let's just call the `this.htmlLengthService.computeHtmlLengthInWords` function directly above.